x/rewards/wasmbinding/types/coins.go

```diff
@@ -0,0 +1,61 @@
+package types
```

Comment:
can't we use: https://github.com/CosmWasm/wasmd/blob/f14c46988aa26d5bb393a200cfb77e5ccc380e68/x/wasm/types/types.go#L291